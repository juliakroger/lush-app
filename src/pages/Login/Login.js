import React, { useState } from 'react'

import { LoginForm } from '../../containers'
import Image from '../../images/lush-login.jpg'
import styles from './Login.module.scss'

const Login = () => {
  const [email, setEmail] = useState(null)
  const [password, setPassword] = useState(null)

  return (
    <div className={styles.loginContainer}>
      <img src={Image} alt='lush' className={styles.image} />
      <LoginForm
        email={email}
        setEmail={setEmail}
        password={password}
        setPassword={setPassword}
        submit={() => {}}
      />
    </div>
  )
}

export default Login
