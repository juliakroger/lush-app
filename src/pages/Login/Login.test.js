import React from 'react'
import { shallow } from 'enzyme'

import Login from './Login'

describe('Pages - Login', () => {
  it('should render correctly', () => {
    const component = shallow(<Login />).dive
    expect(component).toMatchSnapshot()
  })
})
