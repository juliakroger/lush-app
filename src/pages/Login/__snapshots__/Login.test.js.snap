// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages - Login should render correctly 1`] = `[Function]`;
